src/main/java/bhs/devilbotz/Constants.java

```diff
@@ -17,11 +17,32 @@ public final class Constants {
 
   public static final int JOYSTICK_PORT = 0;
 
-  public static final double BALANCE_P = 0.07; // TODO: Get correct values
+  public static final double BALANCE_P = 0.39941; // TODO: Get correct values
   public static final double BALANCE_I = 0.0; // TODO: Get correct values
-  public static final double BALANCE_D = 0.01; // TODO: Get correct values
+  public static final double BALANCE_D = 0.00; // TODO: Get correct values
 
   public static class OperatorConstants {
     public static final int DRIVER_CONTROLLER_PORT = 0;
   }
+
+  public static class DriveConstants {
+    public static final double maxSpeed = 2.447; // meters per second TODO: Measure/SysID
+    public static final double maxAngularSpeed =
+        2 * Math.PI; // one rotation per second TODO: Measure
+
+    public static final double trackWidth = 0.555; // meters, robot width TODO: Measure
+    public static final double wheelRadius =
+        0.0762; // meters (Andymark am-0940b "High Grip Wheels, 6")
+    public static final int encoderResolution = 4096; // CTRE Magnetic Encoder
+
+    // Drive PID values TODO: Tune
+    public static final double driveP = 3.5725;
+    public static final double driveI = 0.0;
+    public static final double driveD = 0.0;
+
+    // Feedforward constants TODO: Tune
+    public static final double driveFFS = 0.94143;
+    public static final double driveFFV = 2.3803;
+    public static final double driveFFA = 0.48128;
```

Comment:
Hi Parker, you need to change these to google style constants.  E.g. max_speed --> MAX_SPEED

Reply:
Already done